src/prediqt.ts

```diff
@@ -71,7 +70,13 @@ export class Prediqt {
         this.eosioContract = EOSIO_CONTRACT;
         this.eosioMultiSigContract = EOSIO_MULTISIG_CONTRACT;
         this.rpc = new JsonRpc(nodeEndpoint, { fetch: fetch as any });
```

Comment:
I think we should set nodeEndpoint part of the apiData.

customApi will contains a public rpc property that we should use.

nodeEndpoint must only be set if we are passing parameters to create api & rpc inside of prediqt